src/Main.java

```diff
@@ -48,23 +47,24 @@ public static void main(String[] args) {
         System.out.println(frog);
         System.out.println("______");
 
+        System.out.println("Задача 6 и 7");
         var boxer1Weight = 78.6;
         var boxer2Weight = 87.2;
         var commonWeight = boxer1Weight + boxer2Weight;
-        System.out.println("Вывод данных задач 6 и 7");
         System.out.println("Общий вес боксеров равен " + commonWeight + "кг");
         var weightDifference = boxer2Weight - boxer1Weight;
         System.out.println("Разница в весе боксеров равна " + weightDifference + "кг");
         var modulo = boxer2Weight % boxer1Weight;
         System.out.println("Остаток от деления веса одного боксера на вес другого равен " + modulo);
         System.out.println("______");
 
+        System.out.println("Задача 8");
         var totalWorktime = 640;
         var singleWorkertime = 8;
-        var totalWorkers = (totalWorktime / singleWorkertime);
-        System.out.println("Вывод данных задачи 8");
+        var totalWorkers = totalWorktime / singleWorkertime;
+        var totalWorkersnew = totalWorkers + 94;
```

Comment:
```suggestion
        totalWorkers = totalWorkers + 94;
```
зачем создавать новую переменную?